src/lib.rs

```diff
@@ -106,6 +109,72 @@ pub fn copy_decode<R: Read, W: Write>(source: R, mut destination: W) -> io::Resu
     Ok(())
 }
 
+/// Find the size in bytes of uncompressed data from xz file.
+pub fn uncompressed_size<R: Read + Seek>(mut source: R) -> io::Result<u64> {
```

Comment:
And, it may need to add `bindgen` feature flag.
```suggestion
#[cfg(feature = "bindgen")]
pub fn uncompressed_size<R: Read + Seek>(mut source: R) -> io::Result<u64> {
```